ts/src/transports/Transport.ts

```diff
@@ -64,10 +71,14 @@ export class DefaultTransportFactory {
       return mozXhrRequest;
     }
 
+    if (XMLHttpRequest.prototype.hasOwnProperty("overrideMimeType")) {
```

Comment:
Preferring XHR with overrideMimeType over ms-stream due to 16MB limitation with ms-stream